apps/frontend/src/pages/tasks/tasks.tsx

```diff
@@ -19,21 +23,83 @@ const Tasks: React.FC = () => {
 		};
 	});
 
+	const [mode, setMode] = useState<ValueOf<typeof TasksMode>>(
+		TasksMode.CURRENT,
+	);
+
 	useEffect(() => {
 		void dispatch(taskActions.getCurrentTasks());
 	}, [dispatch]);
 
+	useEffect(() => {
+		if (mode === TasksMode.CURRENT) {
+			void dispatch(taskActions.getCurrentTasks());
+		}
+
+		if (mode === TasksMode.PAST) {
+			void dispatch(taskActions.getPastTasks());
+		}
+	}, [dispatch, mode]);
+
+	const handleModeToggle = useCallback(() => {
+		setMode((previousState) => {
+			return previousState === TasksMode.CURRENT
+				? TasksMode.PAST
+				: TasksMode.CURRENT;
+		});
+	}, []);
+
+	const handleSkip = useCallback(
+		(id: number): void => {
+			void dispatch(
+				taskActions.update({ data: { status: TaskStatus.SKIPPED }, id }),
```

Comment:
Do we need to send it inside of `data`? Why not just `{ status: TaskStatus.SKIPPED }`?